packages/components/src/Loading/LoadingSpinner/LoadingSpinner.module.css

```diff
@@ -0,0 +1,28 @@
+.loadingSpinner {
+  --loading-spinner-size: var(--spacing-48);
+
+  display: flex;
+  width: var(--loading-spinner-size);
+  height: var(--loading-spinner-size);
+}
+
+.loadingSpinner.xs {
+  --loading-spinner-size: var(--spacing-16);
```

Comment:
Given spinners are closer to images and icons, we should probably actually use `px` values for them as they don't need to scale with increased font size (have been considering that spacing probably makes sense to be too in most cases, but that's a different problem).

Reply:
Makes sense to me, will make the change 👍